Shared/Samples/Query related features/QueryRelatedFeaturesView.swift

```diff
@@ -0,0 +1,205 @@
+            url: .alaskaParksFeatureServer.appending(component: "2")
+        )
+        map.addTables([preservesFeatureTable, speciesFeatureTable])
+        
+        return map
+    }()
+    
+    /// An "Alaska National Parks" feature layer.
+    @State private var parksFeatureLayer = FeatureLayer(
+        featureTable: ServiceFeatureTable(
+            url: .alaskaParksFeatureServer.appending(component: "1")
+        )
+    )
+    /// The text describing the loading status of the sample.
+    @State private var loadingStatusText: String?
+    
+    /// The point on the screen where the user tapped.
+    @State private var tapPoint: CGPoint?
+    
+    /// The results from querying related features.
+    @State private var queryResults: RelatedFeatureQueryResults?
+    
+    /// The error shown in the error alert.
+    @State private var error: Error?
+    
+    var body: some View {
+        GeometryReader { geometryProxy in
+            MapViewReader { mapViewProxy in
+                MapView(map: map)
+                    .onSingleTapGesture { screenPoint, _ in
+                        tapPoint = screenPoint
+                    }
+                    .task(id: tapPoint) {
+                        guard let tapPoint else { return }
+                        parksFeatureLayer.clearSelection()
+                        
+                        do {
+                            // Identifies the tapped screen point.
+                            loadingStatusText = "Identifying feature…"
+                            defer { loadingStatusText = nil }
+                            
+                            let identifyResult = try await mapViewProxy.identify(
+                                on: parksFeatureLayer,
+                                screenPoint: tapPoint,
+                                tolerance: 12
+                            )
+                            
+                            // Selects the first feature in the result.
+                            guard let parkFeature = identifyResult.geoElements.first
+                                    as? ArcGISFeature else { return }
+                            parksFeatureLayer.selectFeature(parkFeature)
+                            
+                            // Queries for related features of the identified feature.
+                            loadingStatusText = "Querying related features…"
+                            
+                            let parksFeatureTable = parksFeatureLayer.featureTable as! ServiceFeatureTable
+                            let queryResults = try await parksFeatureTable.queryRelatedFeatures(
+                                to: parkFeature
+                            )
+                            self.queryResults = RelatedFeatureQueryResults(results: queryResults)
+                        } catch {
+                            self.error = error
+                        }
+                    }
+                    .popover(
```

Comment:
Based on this [comment](https://github.com/Esri/arcgis-maps-sdk-swift-samples/pull/404#discussion_r1597279885), I thought it might be good to show this functionality somewhere in the Sample Viewer. I don't think we will ever build a dedicated sample for this since we generally don't have SwiftUI specific samples. Also, the popover arrow helps imply that the displayed list is related to the selected feature. I don't feel super strongly about it, though, and will remove the `GeometryReader` if you still think this is still unneeded. Thoughts?

Reply:
I would personally prefer a sheet, but no I don't feel super strongly about it.

Reply:
Up to you.

Reply:
I'm not sure. If @yo1995 or @mhdostal don't have opinions, I'll probably just leave as is.

Reply:
I could go either way.